themes/default-bootstrap/product.tpl

```diff
@@ -464,13 +464,12 @@
 			</section>
 			<!--end Data sheet -->
 		{/if}
-		{if $product->description}
+		{if isset($product) && $product->description}
```

Comment:
Hi,

Didin't you mean isset($product->description) ? Nowhere in this tpl is tested  $product.

Regards